pulp_python/app/pypi/views.py

```diff
@@ -110,6 +121,19 @@ def initial(self, request, *args, **kwargs):
         else:
             self.base_content_url = BASE_CONTENT_URL
 
+    # Mocking NamedModelViewSet to get PyPI APIs to support RBAC access polices
+    @classmethod
+    def view_name(cls):
+        return "-".join(cls.endpoint_pieces())
+
+    @classmethod
+    def urlpattern(cls):
+        return "/".join(cls.endpoint_pieces())
+
+    @classmethod
+    def endpoint_pieces(cls):
+        return ["pypi", cls.endpoint_name]
+
```

Comment:
Do we still need these?

Reply:
We still need at least `urlpattern`, `AccessPolicyFromDB` uses that to look up the AP.

Reply:
Then we should remove the others. Also please add a docstring that this is for mimicry.